docs/site/Decorators.md

```diff
@@ -141,58 +141,181 @@ For more usage, refer to
+  - type: A [common name of OpenAPI primitive data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types). 
+  - name: Name of the parameter. It should be a `string`.
+
+  A list of available shortcuts for `query` can be found in [API document](http://apidocs.loopback.io/@loopback%2fopenapi-v3/#param.query),
+  along with the shortcuts for `path` and `header`.
+
+  An equivalent example using the shortcut decorator would be:
+  
+  ```ts
+    class MyController {
+      @get('/Pets/{category}')
+      list(
+        @param.path.string('category') category: string,
+        @param.query.number('pageSizes') pageSize?: number,
+      ) {}
+    }
+  ```
 
-```ts
-// example 2: decorator `@param` applied on parameter level
-class MyController {
-  @get('/')
-  list(
-    @param(offsetSpec) offset?: number,
-    @param(pageSizeSpec) pageSize?: number,
-  ) {}
-}
-```
+  You can find specific use cases in [Writing Controller methods](Controllers.md#writing-controller-methods)
+
+  *The parameter location cookie is not supported yet, see*
+  *https://github.com/strongloop/loopback-next/issues/997*
 
-In the first example, we apply multiple decorators to a single declaration. The
-order in which the decorators are called is important because a `@param`
-decorator must be applied after an operation decorator. To learn more about
-TypeScript decorator composition, refer to
-[TypeScript Decorator Documentation](https://www.typescriptlang.org/docs/handbook/decorators.html)
+### RequestBody Decorator
 
-Please note method level `@param` and parameter level `@param` are mutually
-exclusive, you can not mix and apply them to the same parameter.
+  Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/0739ffcfe3ef50e0bfd86055c0f4e29fd6925be0/packages/openapi-v3/src/request-body-decorator.ts#L20-L79)
 
-You can also use this pattern to make it quicker to define params:
-`@param.${in}.${type}(${name})`
+  `@requestBody()` is applied to a controller method parameter to generate OpenAPI requestBody specification for it.
+  
+  *Only one parameter can be decorated by `@requestBody` per controller method.*
 
-- in: one of the following values: `query`, `header`, `path`, `formData`, `body`
-- type: one of the following values: `string`, `number`, `boolean`, `integer`
-- name: a `string`, name of the parameter
+  A typical [OpenAPI requestBody specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#requestBodyObject)
+  contains properties `description`, `required`, and `content`:
+
+  ```ts
+    requestBodySpec: {
+    description: 'a user',
+    required: true,
+    content: {
+      'application/json': {...schemaSpec},
+      'application/text': {...schemaSpec},
+    },
+  }
+  ```
+
+  In order to use 'requestBody', the parameter type it's decorating needs to have its model decorated with '@model' and '@property'
```

Comment:
nitpick: suggest to use backtick ` for @model and @property instead of single quote.

Reply:
good catch!